api/src/main/java/module-info.java

```diff
@@ -745,10 +747,11 @@
  *
  * <pre>
  * // Query by Method Name
- * Vehicle[] findByMakeAndModelAndYear(String makerName, String model, int year, Sort&lt;?&gt;... sorts);
+ * Vehicle[] findFirst50ByMakeAndModelAndYear(String makerName, String model, int year, Sort&lt;?&gt;... sorts);
  *
  * // parameter-based conditions
  * &#64;Find
+ * &#64;First(50)
  * Vehicle[] searchFor(String make, String model, int year, Sort&lt;?&gt;... sorts);
```

Comment:
This is the section that compares Query by Method Name with Parameter-based find